src/plugins/intel_cpu/src/node.h

```diff
@@ -41,6 +41,8 @@
 #include "nodes/executors/mvn_list.hpp"
 #include "nodes/executors/executor.hpp"
 
+#define THROW_CPU_NODE_ERR(...) OPENVINO_THROW(getTypeStr(), " node with name '", getName(), "' ", __VA_ARGS__)
```

Comment:
There are just about 10 nodes with such definition. Most of nodes have `errorPrefix` member. In my opinion it more convenient to define it in one place:
- It will be easy to modify that if OPENVINO_THROW will be changed again,
- No need to define macros or errorPrefix in each node.